README.md

```diff
@@ -1 +1,57 @@
-# curriculum-backend-readings
\ No newline at end of file
+# Backend Curriculum Readings
+This serves as a content monorepo for all the readings of the backend bootcamp curriculum developed by Re:Coded. Content from this repo will be transferred to the course on Canvas LMS. Each markdown file inside the module folders and sub-folders corresponds to a reading on the LMS.
+
+## Folder Structure
+```
+.
+│   README.md                       // We are here   
+│
+└───module1-introduction-to-backend
+|   └───assets                      // common assets folder for module
+│   └───r1-introduction-to-backend
+|       | README.md                 // reading content goes here
+│   └───r1.1-beginners-guide-nodejs-npm
+|       | README.md
+|   └───r2-http-and-rest
+|       | README.md
+|   └───r2.1-diving-into-rest-apis
+|       | README.md
+|   └───...                         // and so on
+|
+└───module2-database-technologies
+|   └───assets                      // common assets folder for module
+│   └───r1-introduction-to-databases
+|       | README.md                 // reading content goes here
+│   └───r2-introduction-to-relational-databases
+|       | README.md
+|   └───r2.1-relational-database-structure
+|       | README.md
+|   └───...                         // and so on
+|
+└───...                             // and so on
+```
+
+## Folder naming Conventions
+
+- For every new reading, name the folder as: `r[number]-name-of-reading`. For example, `r1-introduction-to-backend` or `r2.1-relational-database-structure`.
```

Comment:
Would it be better if we defined a meta file inside each to define the type and identifier for each lesson rather than embedding it into the name?

Reply:
Yes we will definitely have a meta file in each lesson folder, as it will be required for the Github to Canvas migration tool.
The reason for prefixing r and number in the folder name is mainly to maintain the order of folders for readability and flow.

Reply:
i think parsing the name for the mapping is better. there are a lot of things that can go out of sync with a metafile, for example if someone renames a file without updating the metafile then things will go awry

Reply:
Noted, will look into both options (meta file and parsing folder name) in more detail once we have confirmation from Flatiron School for the Canvas API access.
Till then are we aligned on adding the numbers to folder names to maintain order of folders for our readability?
Can I get both your 👍🏼 to merge this PR?